superset-frontend/src/components/CopyToClipboard/CopyToClipboard.stories.tsx

```diff
@@ -49,6 +49,7 @@ InteractiveCopyToClipboard.args = {
   text: 'http://superset.apache.org/',
   wrapped: true,
   tooltipText: 'Copy to clipboard',
+  hideTooltip: false,
```

Comment:
```suggestion
  hideTooltip: true,
```
I think this would be set to `true` here, right?